examples/llama/tests/utils.py

```diff
@@ -0,0 +1,15 @@
+import importlib
+import sys
+from pathlib import Path
+
+
+def set_system_path():
```

Comment:
won't be needed anymore with the `__init__.py`

Reply:
@yardenas okay I am very sorry, it was not a good idea as it break some stuff. Could you revert to your old way of doing please ?

Reply:
yes ofc. done